tests/Dfe.EarlyYearsQualification.UnitTests/ViewComponents/FooterLinksViewComponentTests.cs

```diff
@@ -49,12 +55,13 @@ public async Task InvokeAsync_ContentServiceReturnsNull_ReturnsEmptyNavigationLi
         var footerLinksViewComponent = new FooterLinksViewComponent(_mockContentService.Object, _mockLogger);
         var result = await footerLinksViewComponent.InvokeAsync();
 
-        Assert.IsNotNull(result);
+        result.Should().NotBeNull();
+
         var model = (result as ViewViewComponentResult)?.ViewData?.Model;
-        Assert.IsNotNull(model);
+        model.Should().NotBeNull();
 
         var data = model as List<NavigationLink>;
-        Assert.IsNull(data);
+        data.Should().BeNull(); // question: should data instead be an empty list?

```

Comment:
In response to your question, yes possibly. This may change however with the work Dan is doing

Reply:
Yes, I often find when writing tests questions like this crop up, I'm not suggesting we change it as part of this PR.